qemu/tests/block_copy.py

```diff
@@ -136,6 +136,40 @@ def is_cancelled():
         if self.vm.monitor.protocol == "qmp":
             self.vm.monitor.clear_event("BLOCK_JOB_CANCELLED")
 
+    def is_paused(self):
+        """
+        Return block job paused status.
+        """
+        paused = self.get_status().get("paused")
+        offset_p = self.get_status().get("offset")
+        time.sleep(random.uniform(1, 3))
+        offset_l = self.get_status().get("offset")
+        paused &= offset_p == offset_l
```

Comment:
@xutian , operator '==' has higher priority than &, so we should not need () here.  I've tested it, it works correctly.